cordova-lib/src/plugman/registry/registry.js

```diff
@@ -236,15 +231,22 @@ function initSettings() {
     module.exports.settings =
     rc('plugman', {
         cache: plugmanCacheDir,
-        registry: 'http://registry.cordova.io',
+        registry: registryURL,
         logstream: fs.createWriteStream(path.resolve(plugmanConfigDir, 'plugman.log')),
         userconfig: path.resolve(plugmanConfigDir, 'config'),
         'cache-min': oneDay
     });
+
+    // if npm is true, use npm registry. 
+    // ~/.plugman/config overides the above config if it exists. 
```

Comment:
I think custom registry should still work. 

Ex: `plugman config set registry "MYSPECIALREGISTRY"`
This updates the registry field in `~/.plugman/config`

By the time the code hits line 243, the registry value equals the registry value from config. If npm is true, then it would set it to use npm. But this is currently set to happen on the fail (fallback). 

So it would hit custom registry and then fallback to npm.